packages/components/src/disabled/index.js

```diff
@@ -103,6 +115,8 @@ function Disabled( { className, children, ...props } ) {
 	);
 }
 
+Disabled.defaultProps = defaultProps;
```

Comment:
Nitpick: `defaultProps` on function component will likely be deprecated in the future. Prefer just use the default value syntax directly on props spread instead.

Reply:
Good call 👍  Updated in https://github.com/WordPress/gutenberg/pull/26730/commits/44611984c449230173f04ffcf234567e5c8e5ce0